mailpoet/lib/Analytics/Reporter.php

```diff
@@ -108,6 +108,7 @@ public function getData() {
       'MailPoet Premium version' => (defined('MAILPOET_PREMIUM_VERSION')) ? MAILPOET_PREMIUM_VERSION : 'N/A',
       'Total number of subscribers' => $this->subscribersFeature->getSubscribersCount(),
       'Sending Method' => isset($mta['method']) ? $mta['method'] : null,
+      "Send all site's emails with" => $this->settings->get('send_transactional_emails') ? 'current sending method' : 'default WordPress sending method',
```

Comment:
I noticed that there doesn't appear to be an integration test for this class. Is that something that would be worth adding as part of this PR?